lib/SubscriptionsManager.js

```diff
@@ -54,9 +55,39 @@ SubscriptionsManager.prototype.set = function (entryId, email) {
     return this._get(entryId).then(list => {
       if (!list) {
         queue.push(new Promise((resolve, reject) => {
-          this.mailAgent.lists().create({
+          let payload = {
             address: listAddress
-          }, (err, result) => {
+          }
+          /*
+           * Only allow authenticated users to post to the list. This is the default, but let's
+           * explicitly set it in case the default changes.
+           */
+          payload.access_level = 'readonly'
+
+          /*
+           * Restricting replies to "sender" (as opposed to all members of the list) would seem
+           * to be the safest and most appropriate option for a list meant to receive
+           * notifications.
+           */
+          payload.reply_preference = 'sender'
+
+          const entryName = options.parentName
+          if (typeof entryName !== 'undefined') {
+            /*
+             * Set a name and description on the created list to aid in identification and
+             * troubleshooting, as the automatically-generated list address is an obfuscated
+             * hash value.
+             */
+            payload.name = entryName
+            /*
+             * For the description, include the elements that are used to generate the list
+             * address hash value.
+             */
+            payload.description = 'Subscribers to ' + entryId +
+              ' (' + this.parameters.username + '/' + this.parameters.repository + ')'
```

Comment:
```suggestion
          payload.access_level = 'readonly'
          payload.reply_preference = 'sender'

          const entryName = options.parentName
          if (entryName) {
            payload.name = entryName
            payload.description = `Subscribers to ${entryId} (${this.parameters.username}/${this.parameters.repository})`
```

nit: Can use fstring for description instead of string concatenation here